core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -182,6 +183,8 @@ public Options fetch(ColumnType... colsToFetch) {
           case SCANS:
             families.add(ScanFileColumnFamily.NAME);
             break;
+          case SUSPEND:
+            families.add(SuspendLocationColumn.SUSPEND_COLUMN.getColumnFamily());
```

Comment:
This needs a break statement to avoid unintentional fall-through:

```suggestion
            families.add(SuspendLocationColumn.SUSPEND_COLUMN.getColumnFamily());
            break;
```

Reply:
Thanks good catch.